projects/driver_controls/src/main.c

```diff
@@ -1,56 +1,80 @@
+
```

Comment:
Probably don't want to merge this test code into `main`... Maybe make a functional test for it instead?